OnlineDB/SiStripConfigDb/src/SiStripDbParams.cc

```diff
@@ -311,7 +311,8 @@ void SiStripDbParams::print(std::stringstream& ss) const {
 
   if (!usingDbCache_) {
     if (usingDb_) {
-      ss << " Database account (ConfDb)  : " << confdb_ << std::endl;
+//      ss << " Database account (ConfDb)  : " << confdb_ << std::endl;
```

Comment:
@quinnanm can you remove the commented line ?

Reply:
yes!